packages/api/tests/endpoints/test-rules.js

```diff
@@ -287,13 +406,22 @@ test('POST creates a rule that is enabled by default', async (t) => {
     .send(newRule)
     .expect(200);
 
+  const fetchedPostgresRecord = await t.context.dbClient.queryBuilder()
+    .select()
+    .table(tableNames.rules)
+    .where({ name: newRule.name })
+    .first();
+
+  t.true(fetchedPostgresRecord.enabled);
   t.is(response.body.record.state, 'ENABLED');
 });
 
 test('POST returns a 409 response if record already exists', async (t) => {
-  const newRule = fakeRuleFactoryV2();
+  const { newRule } = t.context;
+  newRule.createdAt = Date.now();
+  newRule.updatedAt = Date.now();
 
-  await ruleModel.create(newRule);
+  await ruleModel.create(newRule, newRule.createdAt, newRule.updatedAt);
```

Comment:
Is setting the `createdAt` and `updatedAt` properties important for this test?

Reply:
Negatory, removing